firestore/cloud-client/snippets.py

```diff
@@ -947,3 +947,11 @@ def update_document_increment(db):
 
     washington_ref.update({"population": firestore.Increment(50)})
     # [END fs_update_document_increment]
+
+def list_document_subcollections(db):
```

Comment:
Could you add a test for this snippet? https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/firestore/cloud-client/snippets_test.py

Reply:
Yup, good call.  Added.